eng/Versions.props

```diff
@@ -13,8 +13,8 @@
   <PropertyGroup>
     <MicrosoftCodeAnalysisCSharpVersion>2.1.0</MicrosoftCodeAnalysisCSharpVersion>
     <MicrosoftDotNetCliUtilsVersion>1.0.1</MicrosoftDotNetCliUtilsVersion>
-    <NuGetPackagingCoreVersion>4.3.0-rtm-4324</NuGetPackagingCoreVersion>
```

Comment:
Are these ever different? 
NuGet library versions don't play well together, especially because ProjectModel depends on Packaging. 

https://user-images.githubusercontent.com/2878341/50718454-9ed39e00-1044-11e9-8516-2581366fe168.png